src/App.tsx

```diff
@@ -17,18 +40,41 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                filterBySelect={filterBySelect}
+                onSetQuery={setQuery}
+                onSetFilterBySelect={setFilterBySelect}
+              />
             </div>
 
-            <div className="block">
-              <Loader />
-              <TodoList />
-            </div>
+            {errorGetTodos || (
+              <span>
+                {errorGetTodos}
+              </span>
+            )}
+
+            {allTodos.length === 0 || errorGetTodos
+              ? <Loader />
```

Comment:
Don't show the 
![Screenshot 2023-04-06 at 14 26 05](https://user-images.githubusercontent.com/54940595/230363483-4d843433-2e7f-43db-9a7a-52974ae834e8.png)
loader on error.